scripts/load-versions.js

```diff
@@ -5,7 +5,16 @@
 const fs = require('fs')
 const nodeVersionData = require('node-version-data')
 
-module.exports = nodeVersionData
+module.exports = function (callback) {
+  nodeVersionData((err, versions) => {
+    if (err) {
+      return callback(err)
+    }
+
+    versions.forEach((v) => v.lts === false && delete v.lts)
```

Comment:
no, I added this exactly because it didn't ignore it and I ended up with a bunch of `false`s in the column